binding/SkiaSharp/SKColorSpaceStructs.cs

```diff
@@ -135,6 +171,35 @@ public SKColorSpaceTransferFn (float g, float a, float b, float c, float d, floa
 		public readonly float[] Values =>
 			new[] { fG, fA, fB, fC, fD, fE, fF };
 
+
+		public readonly void GetValues (Span<float> values)
+		{
+			if (values.Length != 7)
+				throw new ArgumentException ("The values must have exactly 7 items, one for each of [G, A, B, C, D, E, F].", nameof (values));
+
+			values[0] = fG;
+			values[1] = fA;
+			values[2] = fB;
+			values[3] = fC;
+			values[4] = fD;
+			values[5] = fE;
+			values[6] = fF;
+		}
+
+		public void SetValues (ReadOnlySpan<float> values)
+		{
+			if (values.Length != 8)
```

Comment:
why is this 8 instead of 7?

Reply:
Good catch. I must've copy-pasted from above and forgot to change it.